easybuild/easyblocks/g/gcc.py

```diff
@@ -618,6 +619,52 @@ def post_install_step(self, *args, **kwargs):
             else:
                 raise EasyBuildError("Can't link '%s' to non-existing location %s", target, os.path.join(bindir, src))
 
+        # Rename include-fixed directory which includes system header files that were processed by fixincludes,
```

Comment:
Should we make this block specific to certain versions of GCC, or are we expecting them all to work okay with this level of modification?

Reply:
To which versions should it be specific then? It applies to any version, no?

Reply:
I have no idea. I was more concerned with limiting scope for such a change. Do we know for what versions the change was done by Red Hat? The package we got linked was created for 9.*, so that's fairly recent. We know it's a problem on 8.3, but do we know if 7.* is affected?

Reply:
I see `include-fixed` even as far back as GCC 4.9.2, so I don't think there's a point in adding a version check...

Also, looks like Red Hat has been trashing the stuff in `include-fixed` for a while, see https://git.centos.org/rpms/devtoolset-3-gcc/blob/611c22b0e52a80f78ab2682e7ab33b1e9b908f46/f/SPECS/gcc.spec#_1531, which goes back as far as Oct'14 (GCC 4.9.2)

Reply:
Well, that's terrifying. No version check it is. :-)